### PR TITLE
fix some small visual bugs with configurable table

### PR DESCRIPTION
**Pull request by rossedfort**
## Description & motivation 💭 
fix some small visual bugs with configurable table
- properly display batch actions when there are no columns pinned
- hide resize handle when batch actions are disabled and there are no pinned columns
#### before
Screen Shot 2023-04-20 at 3 45 14 PM
Screen Shot 2023-04-20 at 3 45 38 PM

#### after
Screen Shot 2023-04-20 at 3 44 52 PM
Screen Shot 2023-04-20 at 3 45 56 PM

### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
Run oss locally and visually verify changes
## Checklists
N/a
### Issue(s) closed 
N/a
## Docs
N/a

### CODE REVIEW

File: src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte [hunk 1]
```diff
@@ -420,6 +421,10 @@
     @apply z-10 bg-primary w-0 border-r-[3px] border-primary cursor-col-resize;
   }
 
+  .resizer.no-columns-pinned:not(.batch-actions-enabled) {
```
Comment (laurakwhit) [inline src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte:424]: I would say just hide the resizer when batch actions are enabled too
```suggestion
  .resizer.no-columns-pinned{
```

but I think the hover state is maybe a bit of a blocker for now. What do you think?

Screenshot 2023-04-20 at 4 12 01 PM

File: src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte [hunk 2]
```diff
@@ -444,7 +449,7 @@
   }
 
   .workflow-summary-table-wrapper.pinned.no-columns-pinned.batch-actions-visible {
```
Comment (laurakwhit) [inline src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte:451]: We might want to consider a solution for this overlap in the future

Screenshot 2023-04-20 at 4 05 36 PM

**Review by laurakwhit — APPROVED** [review]

File: src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte (hunk 1 above)
Comment (rossedfort) [reply src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte:424]: Hm yeah I think just leaving the divider/resizer is OK for now, but I agree that it'd be nicer to hide it an fix the hover state in this case.

File: src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte (hunk 2 above)
Comment (rossedfort) [reply src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte:451]: I think this is OK given the complexity of this table. Ideally we have less stuff in the table headers IMO 😂

**Pull request merged**